packages/apollo-cli/src/commands/jbrowse/set-config.ts

```diff
@@ -0,0 +1,75 @@
+import * as fs from 'node:fs'
+import {
+  ImportJBrowseConfigChange,
```

Comment:
```suggestion
import type {
  SerializedImportJBrowseConfigChange,
```

Import only the types here, we can't use the actual object in the CLI code due to dependency issues.